packages/nextra/tsup.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'tsup'
+export default defineConfig([{
+  entry: [
+    'src/index.ts',
+    'src/ssg.ts',
+    'src/locales.ts',
+    'src/context.ts'
+  ],
+  format: 'cjs',
+  dts: true,
+  name: 'nextra',
+  external: ['next/server']
+},
+{
+  entry: ['src/loader.ts'],
+  format: 'esm',
+  name: 'nextra-loader',
+  dts: true
+},
+{
+  entry: ['src/compile.ts'],
+  format: 'esm',
+  name: 'compile',
+  dts: true
+}, {
+  entry: ['src/icons.ts'],
```

Comment:
```suggestion
  entry: ['src/icons/index.ts'],
```
and you can remove `src/icons.ts` I guess?

Reply:
we have use icons.ts here since we want it output `icon.mjs` in dist